activestorage/lib/active_storage.rb

```diff
@@ -65,6 +66,13 @@ module ActiveStorage
 
   mattr_accessor :replace_on_assign_to_many, default: false
 
+  mattr_accessor :delivery_methods,        default: {}
+  mattr_accessor :default_delivery_method, default: :redirect
+
+  mattr_accessor :proxy_urls_expire_in, default: 525600.minutes
```

Comment:
Use a more readable time format. 525600 must be some number of hours? Or days? Use that.